src/legacy/core_plugins/kibana/public/dashboard/dashboard_empty_screen.tsx

```diff
@@ -18,17 +18,29 @@
  */
 import React from 'react';
 import { I18nProvider, FormattedMessage } from '@kbn/i18n/react';
-import { EuiIcon, EuiLink } from '@elastic/eui';
+import {
+  EuiIcon,
+  EuiLink,
+  EuiSpacer,
+  EuiPageContent,
+  EuiPageBody,
+  EuiPage,
+  EuiText,
+} from '@elastic/eui';
 import * as constants from './dashboard_empty_screen_constants';
 
-export interface Props {
+export interface DashboardEmptyScreenProps {
   showLinkToVisualize: boolean;
   onLinkClick: () => void;
+  onVisualizeClick?: () => void;
 }
 
-export function DashboardEmptyScreen({ showLinkToVisualize, onLinkClick }: Props) {
+export function DashboardEmptyScreen({
+  showLinkToVisualize,
+  onLinkClick,
+}: DashboardEmptyScreenProps) {
   const linkToVisualizeParagraph = (
-    <p className="linkToVisualizeParagraph">
+    <EuiText data-test-subj="linkToVisualizeParagraph">
```

Comment:
`EuiText` only renders `div`s around everything. Can we leave the `p` tags around content?  (`<EuiText>` can wrap `<p>`.)

Reply:
I ruins the fonts if it's not `EuiText`. I am not quite where/how is the EUI stylesheet loaded. Do you know a way to keep the fonts and not have a `EuiText`?

Reply:
@majagrubic I think what @myasonik is trying to say is that EuiText just provides styling for the text, but you still need to provide the text element like a paragraph `<p>` or heading `<h2>` tag. So in this instance you would keep the `EuiText` but also wrap the FormattedMessage inside a `<p>` tag.

Reply:
🤦‍♀ 🤦‍♀Let's pretend I never said that!